java/core/page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/SetupStepDefs.java

```diff
@@ -2,37 +2,37 @@
 
 import cucumber.api.java.After;
 import cucumber.api.java.Before;
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
 import org.apache.log4j.PropertyConfigurator;
 import org.openqa.selenium.WebElement;
 import org.reflections.Reflections;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import ru.sbtqa.tag.allure.TagAllureReporter;
 import ru.sbtqa.tag.allurehelper.ParamsHelper;
 import ru.sbtqa.tag.pagefactory.PageContext;
 import ru.sbtqa.tag.pagefactory.PageFactory;
 import ru.sbtqa.tag.pagefactory.WebElementsPage;
 import ru.sbtqa.tag.pagefactory.annotations.ElementTitle;
-import ru.sbtqa.tag.pagefactory.support.OnFailureScheduler;
-import ru.sbtqa.tag.qautils.properties.Props;
+import ru.sbtqa.tag.pagefactory.support.properties.Properties;
 import ru.sbtqa.tag.qautils.reflect.ClassUtilsExt;
 import ru.sbtqa.tag.qautils.reflect.FieldUtilsExt;
 import ru.sbtqa.tag.videorecorder.VideoRecorder;
 
-import java.io.File;
-import java.lang.reflect.Field;
-import java.util.*;
-
 public class SetupStepDefs {
 
     private static final Logger LOG = LoggerFactory.getLogger(SetupStepDefs.class);
 
     @Before
     public void setUp() {
 
-        //Apply failure callback
-        TagAllureReporter.applyFailureCallback(OnFailureScheduler.class);
```

Comment:
How will it works with J7 and allure 1 case?

Reply:
support for the allure1 will be implemented in the next release